src/query-builder/InsertQueryBuilder.ts

```diff
@@ -20,7 +24,8 @@ export class InsertQueryBuilder<Entity> extends QueryBuilder<Entity> {
      */
     getQuery(): string {
         let sql = this.createInsertExpression();
-        return sql.trim();
+        // regexp converting multiline query string into single line
```

Comment:
can you please provide more details why we need this?

Reply:
To get more readability I changed sql query from long single line to line with breaks.
It's for what regexp is used.

Reply:
Can you please make another fix of this issue without regexp applied? Im really scary about using regexp in user defined expressions since it can contain string with values which should not be parsed with regexp. Also regexp brings a performance issue with is critical in insertion process when multiple inserts are called.